ruby-challenges.rb

```diff
@@ -21,13 +33,33 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+def delete_vowel(str)
+  p str.delete 'aeiouAEIOU'
+end
 
+delete_vowel 'Rubber Soul'
+delete_vowel 'Sgt Pepper'
+delete_vowel 'Abbey Road'
 
-# -------------------3) Create a method that takes in a string and checks if the string is a palindrome. A palindrome is the same word spelled forwards or backwards. Use the test variables provided.
+# -------------------3) Create a method that takes in a string and checks if the string is a palindrome.
+# A palindrome is the same word spelled forwards or backwards. Use the test variables provided.
 
 is_palindrome1 = 'Racecar'
 # Expected output: 'Racecar is a palindrome'
 is_palindrome2 = 'LEARN'
 # Expected output: 'LEARN is not a palindrome'
 is_palindrome3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
```

Comment:
The logic is great on the method. However, you need to consider all cases to get the expected output for 'Rotator'. Recommend adding a method that will help you evaluate the words regardless if it is capitalized, lowercase, or uppercase.